app/views/clients/_form.html.slim

```diff
@@ -75,27 +61,8 @@
             = f.input :comments
           = f.input :competent_authority
           = f.input :involved_authority
-      .row
-        .col-xs-12
-          h3= t('schedule')
-          .row
-            .col-xs-12
-              table.table.table-striped
-                thead
-                  tr
-                    th= t('when')
-                    - Schedule.days.each do |day|
-                      th.days= t("weekdays.#{day}")
-                tbody
-                  - @client.schedules.group_by(&:time).each do |time, schedules|
-                    tr
-                      td
-                        = t("times.#{time}")
-                      = f.simple_fields_for :schedules, schedules do |schedule|
-                        td
-                          = schedule.input :day, as: :hidden
-                          = schedule.input :time, as: :hidden
-                          = schedule.input :available, as: :boolean, label: false
+
+    = render 'schedules/fields', f: f, participant: @client
```

Comment:
Please change the naming of this partial to the same as the others: `= render 'schedules/form', f: f, participant: @client`